cmd/start.go

```diff
@@ -97,7 +97,7 @@ func storeGenesisData(bz *bazooka.Bazooka, DBI *db.DB, cfg config.Configuration)
 	accountTreeDepth, err := bz.SC.AccountRegistry.DEPTH(&opts)
 	common.PanicIfError(err)
 
-	err = DBI.InitAccountTree(int(accountTreeDepth.Uint64()))
+	err = DBI.InitAccountTree(int(accountTreeDepth.Uint64()) + 1)
```

Comment:
Nice catch here! The DEPTH in the accountTree contract is misleading, created an issue to fix it later https://github.com/thehubbleproject/hubble-contracts/issues/457😓 .

Reply:
This one lead to finding another big issue, so no harm no foul.